relax/module.py

```diff
@@ -216,7 +216,37 @@ def validation_step(self, params, rng_key, batch):
 
 
 # %% ../nbs/04_learning.ipynb 7
-def load_pred_model(data_name: str) -> Tuple[hk.Params, PredictiveTrainingModule]:
+DATASET_NAMES = ["adult","credit","heloc","oulad","student_performance","titanic","german","cancer","spam", "ozone", "qsar", "bioresponse", "churn", "road"]
+
+def load_pred_model(
+    data_name: str # The name of data
+    ) -> Tuple[hk.Params, PredictiveTrainingModule]:
+    """High-level util function for loading trained model."""
+
```

Comment:
I would suggest moving the newly added code into a new function called `download_model`.